login-workflow/src/components/password-strength-checker/password-strength-checker.component.scss

```diff
@@ -0,0 +1,20 @@
+.pxb-auth-password-check {
+    height: auto;
+    ::ng-deep .mat-list-base .mat-list-item .mat-list-item-content {
+        padding: 0;
+    }
+    .validation-message {
+        color: #424e54; //black[500]
+    }
+
+    .validation-message {
+        padding-left: 8px;
+        font-weight: 400;
+        line-height: 1.2;
+    }
+}
+
+mat-icon,
+.validation-message.success {
+    color: #9ca5a9; //gray[300]
```

Comment:
text secondary?

Reply:
> > At this point, angular _really_ discourages package dependencies, so I'd opt for the former.
> 
> Say what? It's pretty hard to build a package without dependencies.

Angular ng-pkgr throws a warning if you ever list something as a dependency; peer-dependencies are a-okay.